dirsrvtests/tests/suites/replication/virtual_attribute_test.py

```diff
@@ -0,0 +1,214 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
```

Comment:
Please, change the year to 2022 :)

Reply:
oops! still live in 2021 huh! fixing it right away!